dataeng/resources/setup-platform-venv-py3.sh

```diff
@@ -3,12 +3,16 @@
 # This version of setup-platform-venv is specifically designed for newer
 # python3 versions of the platform.
 
+
+# Create and activate a virtualenv in shell script
+VENV="venv-${BUILD_NUMBER}"
```

Comment:
We can be more explicit with the naming here. Something like
`PLATFORM_VENV="platform_venv"`

I don't think we need a separate venv per build, unless we also remove the ones from prior runs.
Also, there is only one instance of this job running at any point, so I think just using it with `--clear` should work.

We need to echo it to the file as it is needed for the workers. And I believe we would need an absolute path as
that's how  `VIRTUAL_ENV` behaved.

`echo "PLATFORM_VENV=${WORKSPACE}/${PLATFORM_VENV}" > platform_venv`

Reply:
ok right,
should be echo "PLATFORM_VENV=${WORKSPACE}/${**VIRTUAL_VENV**}" > platform_venv ??

Reply:
Previously it is exporting the path from `VIRTUAL_ENV` not sure where that is coming from but is the current push correct or should it be `echo "PLATFORM_VENV=${WORKSPACE}/${PLATFORM_VENV}" > platform_venv`

@HassanJaveed84

Reply:
Yes, it should be PLATFORM_VENV, VIRTUAL_ENV was coming from shiningpanda I believe.